pkg/test/istio_io/yaml_step.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Istio Authors
```

Comment:
@geeknoid Is Copyright 2019 correct? I originally had this, but there was a PR to remove the year from the copyright for quite a few files so I followed that model.

Reply:
I haven't seen this elsewhere in the codebase ... but I'll defer to @geeknoid

Reply:
Yeah, modern code doesn't need to use a year. It doesn't hurt to have it there, but it's not necessary.

Reply:
Ok, I'll leave it there for now since it's at least somewhat consistent with other code.

We'll probably want to go through the entire repo in the future and scrub the year.